coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,18 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertObjectsEqual(self, observed_result, expected_result):
+        attributes = Result.__compare_fields__
```

Comment:
instead of using only the fields from `Result`, get it from the compared objects below :)